splash/tests/test_execute.py

```diff
@@ -3925,3 +3925,433 @@ def test_key_error(self):
         """)
         self.assertScriptError(resp, ScriptError.SPLASH_LUA_ERROR,
                                message="Unknown key")
+
+
+class HTMLElementTest(BaseLuaRenderTest):
+    def test_select(self):
+        resp = self.request_lua("""
+        function main(splash)
+            assert(splash:go(splash.args.url))
+            assert(splash:wait(0.1))
+
+            local p = splash:select('p')
+            local form = splash:select('form#login')
+            local username = splash:select('input[name="username"]')
+            local password = splash:select('input[name="password"]')
```

Comment:
From this example it looks like element:select() is a good idea - I'd better write `form:select(...)` here, as ideally the lookup should be limited to `form#login`.